### ISSUE TITLE
FindSpot directory missing from repo?

### ISSUE THREAD

**Issue by clayne** [FelixBer/FindSpot#1]
> Copy the entire FindSpot folder from this repository into %PINDIR%/source/tools/

Problem is, said directory doesn't seem to exist. :-)

**Comment by FelixBer** [comment]
Indeed!  It turns out the current approach wont work for Windows, and I need Windows support. A new approach is ready but hasn't been tested yet on Windows. I am on vacation right now, but I promise a full release next week!

Happy to see there is an interest in this. :+1:

**Comment by FelixBer** [comment]
Alright, here we go. Work continues...
